gettsim/transfers/rente.py

```diff
@@ -295,4 +413,447 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the age, at which a women is eligible to claim the full pension
+        (without deductions). This pension scheme allows for early retirement
+        from age 60 with deductions. Hence this threshold is needed as reference
+        for calculating the zugangsfaktor.
+
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    returns the lowest full retirement age for women.
+
+    """
+    if geburtsjahr < 1945:
+        x = geburtsjahr + (geburtsmonat - 1) / 12
+    else:
+        x = geburtsjahr
+
+    out = piecewise_polynomial(
+        x=x,
+        thresholds=ges_rente_params["altersrente_für_frauen"]["thresholds"],
+        rates=ges_rente_params["altersrente_für_frauen"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params["altersrente_für_frauen"][
+            "intercepts_at_lower_thresholds"
+        ],
+    )
+
+    return out
+
+
+def _ges_rente_langjährig_altersgrenze(
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the age, at which a long term insured person (at least 35 years ) is
+        eligible to claim the full pension. This pension scheme allows for early
+        retirement from age 63 with deductions. Hence this threshold is needed as
+        reference for calculating the zugangsfaktor.
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+    ges_rente_vorraus_besond_lang
+        See:func:`ges_rente_vorraus_besond_lang`.
+
+    Returns
+    -------
+    Full retirement age (without deductions) for long term insured.
+    """
+
+    if geburtsjahr < 1951:
+        x = geburtsjahr + (geburtsmonat - 1) / 12
+    else:
+        x = geburtsjahr
+
+    out = piecewise_polynomial(
+        x=x,
+        thresholds=ges_rente_params["altersgrenze_langjährig_versicherte"][
+            "thresholds"
+        ],
+        rates=ges_rente_params["altersgrenze_langjährig_versicherte"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params[
+            "altersgrenze_langjährig_versicherte"
+        ]["intercepts_at_lower_thresholds"],
+    )
+
+    return out
+
+
+def ges_rente_bes_lang_altersgrenze(
+    geburtsjahr: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the threshold from which very long term insured
+        people (at least 45 years) can claim their full pension
```

Comment:
intendation in docstring is in several functions here different than in other modules. Let's stay consistent
```suggestion
    people (at least 45 years) can claim their full pension
```

Reply:
changed